apps/database/supabase/migrations/20260623063745_add_course_test_attempts.sql

```
@@ -0,0 +1,138 @@
+grant update on table "public"."course_test_attempt_answers" to "authenticated";
+
+grant delete on table "public"."course_test_attempt_answers" to "service_role";
+grant insert on table "public"."course_test_attempt_answers" to "service_role";
+grant references on table "public"."course_test_attempt_answers" to "service_role";
+grant select on table "public"."course_test_attempt_answers" to "service_role";
+grant trigger on table "public"."course_test_attempt_answers" to "service_role";
+grant truncate on table "public"."course_test_attempt_answers" to "service_role";
+grant update on table "public"."course_test_attempt_answers" to "service_role";
+
+-- RLS policies
+create policy "Allow select for user who owns attempt"
```

SQLFluff (4.2.2)

[error] Do not use special characters in identifiers.

(RF05)